IKOR-GmbH/sip-framework
# Docs/proxy mock config example update

# Description

Updating proxy mock configuration example. Setting fully working example with placeholder for customization code.

## Type of change

Please delete options that are not relevant.

# How Has This Been Tested?

- Create MockConfiguration class and copy the code from core.md file under Setting mock behavior example.
- Make sure that configuration yml file on class path contains following key "endpoint.out.partner.their-assurance-co.id", and assigned endpoint id to it
- Make sure that camel route contains endpoint defined in previous step
- (optional) Add custom logic in place of /*define mock behavior*/ comment
- Start the application in debug mode with breakpoint inside MockConfiguration#mockProcessorBehavior, and check if the mock is created 

# Checklist:
